### ISSUE TITLE
Radio ICON Button Select Area is too small

### ISSUE THREAD

**Issue by ZaneGeiser** [blackbaud/skyux#2064]
**Describe the bug**
Sky Radio Icon Buttons have a too small of a select area that only covers the top half of the button. This means the clickable area is  half of the button and and wrappers (like tooltips oriented at the bottom of the radio button) are oriented too high.

**Steps to reproduce the behavior**
1. Go to the Radio button example https://developer.blackbaud.com/skyux/components/radio?docs-active-tab=design
2. Open the browser developer tools.
3. Find the <sky-radio ...> tag in the inspector tools.
4. Hover and see that the dimensions for the <sky-radio> tag are 35x19px.  Whereas the dimensions for the <label class="sky-radio-wrapper sky-switch"> underneath are 35x35px.

**Expected behavior**
The dimensions of the <sky-radio> top level object should match that of the underlying displayed button. In my case it is 35x35px. But could be responsively resized based on browser settings.

**Screenshots**
![Screenshot 2024-02-29 at 5 11 32 pm](https://github.com/blackbaud/skyux/assets/57169901/e31ce2f9-699d-4f44-a9d0-8483c4d9d480)
![Screenshot 2024-02-29 at 5 07 52 pm](https://github.com/blackbaud/skyux/assets/57169901/0bc7f1da-f551-4434-84d9-d4f3feae6363)

**Desktop details:**

- OS: Mac OS 14.1.2
- Browser: Chrome
- Browser version: 1.63.162

**Smartphone details:**
NA

**Additional context**

[AB#2843538](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2843538)

**Comment by Blackbaud-SteveBrush** [comment]
Do you mind providing more context around, "like tooltips oriented at the bottom of the radio button"? We were able to click on all areas of the radio icon without issue.

**Comment by ZaneGeiser** [comment]
Sure thing,

My intention is to use popover the [ngbtooltips](https://ng-bootstrap.github.io/#/components/tooltip/api) included with bootstrap to display useful tool tips below the radio buttons. like this:
```
<sky-radio
    icon="exampleIcon"
    ngbTooltip="helpful tooltip text"
    placement="bottom"
/>
```

At the moment, the described issue causes the tooltip display like this:
![Screenshot 2024-03-01 at 9 21 45 am](https://github.com/blackbaud/skyux/assets/57169901/c49c3271-8982-4af2-b757-0ff86f46a7e9)

Ideally, the tooltip would come from the bottom of the button more like so:
![Screenshot 2024-03-01 at 9 22 11 am](https://github.com/blackbaud/skyux/assets/57169901/cbef8264-7bb9-4e48-a97f-6976b60a0375)

**Comment by Blackbaud-SteveBrush** [comment]
https://github.com/blackbaud/skyux/pull/2082

**Comment by Blackbaud-SteveBrush** [comment]
Fixed in `9.29.0` https://github.com/blackbaud/skyux/blob/main/CHANGELOG.md#9290-2024-03-04. Thanks!
